src/libraries/System.Private.CoreLib/src/System/IO/File.cs

```diff
@@ -689,6 +689,63 @@ public static void WriteAllBytes(string path, byte[] bytes)
             RandomAccess.WriteAtOffset(sfh, bytes, 0);
         }
 
+        /// <summary>
+        /// Appends the specified byte array to the end of the file at the given path. 
+        /// If the file doesn't exist, this method creates a new file.
+        /// </summary>
+        /// <param name="path">The path of the file to which the bytes should be appended.</param>
```

Comment:
Follow existing conventions.
```suggestion
        /// <param name="path">The file to append to.</param>
```